src/app/views/collections/create/CollectionCreateController.jsx

```diff
@@ -132,6 +131,23 @@ export class CollectionCreateController extends Component {
             });
     }
 
+    handleCollectionNameValidation = event => {
+        const name = event.target.value.trim();
+        const nameTaken = this.props.collections.some(c => c.name === name);
+
+        if (nameTaken) {
+            const collectionName = {
+                value: name,
+                errorMsg: "A collection with this name already exists",
+            };
+            const newCollectionDetails = {
+                ...this.state.newCollectionDetails,
+                name: collectionName,
+            };
+            this.setState({ newCollectionDetails });
```

Comment:
A bit confused how this comes out to show the error. 

Is there code somewhere which essentially checks for state.newCollectionDetails.errorMsg and if true output the error?

Reply:
Hi @JoshSinghONS , thank you for looks. It would be helpful to leave the mark on the channel so others know you are looking. 
Yes, this all happens in a child component on here: https://github.com/ONSdigital/florence/blob/develop/src/app/views/collections/create/CollectionCreate.jsx#L105